src/data/faq_de.json

```diff
@@ -485,7 +485,8 @@
                         "anchor": "other_platforms",
                         "active": true,
                         "textblock": [
-                            "Es ist im Moment nicht absehbar, wann und ob wir die App in anderen App Stores außer denen von Google oder Apple zur Verfügung stellen können. Die App wird die Apple und Google Exposure Notification APIs verwenden. Die Community kann sich gerne einbringen und die App für andere APIs übernehmen."
+                            "Es ist im Moment nicht absehbar, wann und ob wir die App offiziell in anderen App Stores außer denen von Google oder Apple zur Verfügung stellen können. Die App wird die Apple und Google Exposure Notification Frameworks verwenden. Die Community kann sich gerne einbringen und die App für andere Frameworks übernehmen.",
+                            "Für den F-Droid Store existiert ein inoffizieller Fork der App unter dem Namen <a href='https://codeberg.org/corona-contact-tracing-germany/cwa-android' target='_blank' rel='noopener noreferrer'>Corona Contact Tracing Germany</a>, welcher von der Community betreut wird. Da die Entwicklung des Forks nicht vom Robert-Koch-Institut betreut wird, erhält die im F-Droid Store veröffentlichte App jedoch keinen offiziellen Support."

```

Comment:
"Robert-Koch-Institut" => "Robert Koch-Institut" or use RKI abbreviation from previous suggestion.

Reply:
fixed

Reply:
Actually not fixed, so I have to correct myself. The spelling with one hyphen is the correct one, see https://www.rki.de/DE/Content/Institut/institut_node.html

They changed it in 1994 for whatever reason